src/Roslyn.Suppressions/Roslyn.Suppressions/Logger.cs

```diff
@@ -34,6 +34,11 @@ public void WriteLine(string messageFormat, params object[] args)
         {
             Debug.WriteLine(messageFormat, args);
         }
+
+        public void LogDebug(string messageFormat, params object[] args)
+        {
+            WriteLine(messageFormat, args);
```

Comment:
This would be a functional change, but I think it's ok. We were using Debug.WriteLine in the WriteLine implementation anyway.

Reply:
Maybe prefix the message with DEBUG?

Reply:
Yes, it got lost in the moving, I handled it in the future PR